OpenOversight/app/models.py

```diff
@@ -551,7 +552,7 @@ def verify_password(self, password):
 
     def generate_confirmation_token(self, expiration=3600):
         payload = {"confirm": self.id}
-        return self._jwt_encode(payload, expiration).decode("utf-8")
+        return self._jwt_encode(payload, expiration).decode(model_encoding)
```

Comment:
Used a local constant because using the one in utils would have created a circular dependency.

Reply:
That makes sense. Looking at the utils file, it is a big mess. We should consider breaking that up by functionality or something to avoid circular dependencies. I will create an issue for that

Reply:
Yeah, I was thinking of moving the stuff to a different package, but I felt that would be a much bigger more than necessary. I'll address that issue next.